sysid-application/src/main/native/include/sysid/analysis/AngularDriveSim.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <Eigen/Core>
+#include <units/time.h>
+#include <units/voltage.h>
+
+namespace sysid {
+/**
+ * Simulates an angular drivetrain. This is primarily intended for unit testing
+ * purposes.
+ */
+class AngularDriveSim {
+ public:
+  /**
+   * @param Ks              Static friction gain.
+   * @param Kv              Velocity gain.
+   * @param Ka              Acceleration gain.
+   * @param initialPosition Initial elevator position.
+   * @param initialVelocity Initial elevator velocity.
```

Comment:
Is this linear or angular position/velocity? The model states should be documented.

Reply:
Is the new explanation better?

Reply:
Do Kv and Ka also use angular units here?

Reply:
No, they are linear. Currently, the model takes the linear gains converted into angular gains, and then when generating the data, it spits out the linear gains again (effectively canceling out the conversions). Would it just be better to start with the linear gains and just spit those out?

Reply:
The linear ones are fine. Just document the units.